source/plugin/economy/using.rst

```diff
@@ -0,0 +1,73 @@
+=====================
+Using the Economy API
+=====================
+
+The Economy API unifies all economy plugins under one API. This means any plugin using the Economy API
+will be compatible with all economy plugins that implement the API. This page guides you through the steps of using 
+the Economy API in your own plugin.
+
+Loading the EconomyService
+==========================
+
+In order to utilize the Economy API, you must first load the EconomyService class:
+
+1. Listen to the ``ChangeServiceProviderEvent`` in order to grab an instance of the EconomyService when it is 
+registered.
+
+2. When the event is fired, check if the service added was the ``EconomyService``. If the it was the 
+``EconomyService``, assign it to a variable that can be used to later access the Economy API.
+
+**Example: Loading the EconomyService**
+
+.. code-block:: java
+
+	import org.spongepowered.api.event.Listener;
+	import org.spongepowered.api.event.service.ChangeServiceProviderEvent;
+	import org.spongepowered.api.service.economy.EconomyService;
+
+	private EconomyService economyService;
+	
+	@Listener
+	public void onChangeServiceProvider(ChangeServiceProviderEvent event) {
+		if (event.getService().equals(EconomyService.class)) {
+			Object rawProvider = event.getNewProviderRegistration().getProvider();
+			if (rawProvider instanceof EconomyService) {
```

Comment:
You can directly cast here. The `ServiceManager` ensures that the provider implements the service, through a generic type bound.